compliance/features/basic.feature

```diff
@@ -16,11 +16,15 @@ Feature: Basic package types support
   @3.3.0
   Scenario: Basic content package can be installed
    Given the "basic_content" package is installed
+     And prebuilt detection rules are loaded
```

Comment:
What are these prebuilt rules? Are they needed to install the packages or other rules?

Reply:
Those prebuilt rules are the ones present in the package plus other rules (in total there are more than 1000).